src/init.cpp

```diff
@@ -601,7 +601,7 @@ void CleanupBlockRevFiles()
     // keeping a separate counter.  Once we hit a gap (or if 0 doesn't exist)
     // start removing block files.
     int nContigCounter = 0;
-    BOOST_FOREACH(const PAIRTYPE(std::string, fs::path)& item, mapBlockFiles) {
+    for (const PAIRTYPE(std::string, fs::path)& item : mapBlockFiles) {
```

Comment:
PAIRTYPE only exists in service of BOOST_FOREACH, would it make sense to remove it in this PR as well?